### PR TITLE
Update find-and-fix-issues.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a related content about Proxy calc and its trigger after domain change

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2809

File: microsoft-365/admin/get-help-with-domains/find-and-fix-issues.md [hunk 1]
```diff
@@ -93,4 +93,9 @@ If you've fixed any DNS issues and you're still having trouble, try one of the f
 - People can't get to your website at www.mydomain.com: [Track down website issues](https://docs.microsoft.com/microsoft-365/admin/setup/add-domain)
     
 - You can't update your A record or CNAME record to point to your website: [Update custom DNS records in Microsoft 365](../dns/add-or-edit-custom-dns-records.md)
+
+## Related content
+
+[Audit data on verified domain change](https://docs.microsoft.com/azure/active-directory/reports-monitoring/troubleshoot-audit-data-verified-domain)
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/get-help-with-domains/find-and-fix-issues.md]:
```suggestion
[Troubleshoot: Audit data on verified domain change](https://docs.microsoft.com/azure/active-directory/reports-monitoring/troubleshoot-audit-data-verified-domain)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@pebaum Please merge this PR. Thanks

**Pull request merged**